app/src/main/kotlin/com/ark/globe/fragments/locations/LocationsFragment.kt

```diff
@@ -19,13 +19,19 @@ import com.ark.globe.R
 import com.ark.globe.adapters.LocationsAdapter
 import com.ark.globe.coordinates.Coordinates
 import com.ark.globe.coordinates.Location
+import com.ark.globe.databinding.LocationInputBinding
 import com.ark.globe.repositories.Repository
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
 class LocationsFragment: Fragment() {
 
     private val activity: AppCompatActivity by lazy{
         requireActivity() as AppCompatActivity
     }
+
+    private var _binding: LocationInputBinding? = null
+    private val binding get() = _binding!!
```

Comment:
You can replace it with this [library](https://github.com/androidbroadcast/ViewBindingPropertyDelegate) 
`private val binding by viewBinding(FragmentLocationsBinding::bind)`

Reply:
Need dependency without reflection 
`implementation 'com.github.kirich1409:viewbindingpropertydelegate-noreflection:1.5.6'`